build/UpdateNuGetModuleManifest.ps1

```diff
@@ -0,0 +1,40 @@
+param (
+    [Parameter(Mandatory = $True)]
+    [string] $NuGetPackageManagementPowerShellCmdletsFilePath,
+    [Parameter(Mandatory = $True)]
+    [string] $ManifestModuleSourceFilePath,
+    [Parameter(Mandatory = $True)]
+    [string] $ManifestModuleDestinationFilePath)
+
+$assembly = [System.Reflection.Assembly]::LoadFile($NuGetPackageManagementPowerShellCmdletsFilePath)
+
+[string] $manifestModuleFileContents = [System.IO.File]::ReadAllText($ManifestModuleSourceFilePath)
+
+$from = 'NuGet.PackageManagement.PowerShellCmdlets.dll'
+$to = $assembly.FullName
+
+Write-Host "Source file:  $ManifestModuleSourceFilePath"
+Write-Host "Destination file:  $ManifestModuleSourceFilePath"
+Write-Host "Replacing `'$from`' with `'$to`'"
+
+$startIndex = $manifestModuleFileContents.IndexOf($from)
+
+If ($startIndex -eq -1)
```

Comment:
As part of our "style" discussions we should also include powershell into the list :)
For example do we use `If` or `if`. 

Personally I think keywords should not be capitalized. 
Powershell is the exception that allows you to capitalize them.